gpu/Makefile

```diff
@@ -85,6 +85,9 @@ CUOBJ=      impl/BroadcastSum.o \
 $(LIBNAME).a: $(CPPOBJ) $(CUOBJ)
 	ar r $@ $^
 
+$(LIBNAME).$(SHAREDEXT): $(CPPOBJ) $(CUOBJ)
```

Comment:
Just a quick heads up, the `clean` target in 119 should probably consider this file as well.